exonum-java-binding/core/src/main/java/com/exonum/binding/runtime/ServiceRuntimeBootstrap.java

```diff
@@ -20,12 +20,16 @@
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import com.google.inject.Module;
+import com.google.inject.Stage;
 
 /**
  * A bootstrap loader of the service runtime.
  */
 final class ServiceRuntimeBootstrap {
 
+  // TODO: Shall it be configurable (Stage.DEV in tests, Stage.PRODUCTION in prod)?
```

Comment:
Where other than in `ServiceRuntimeTest` would we want to use `Stage.DEVELOPMENT`?

Reply:
In native ITs, I think.

Reply:
There aren't many of them either.